odoo/addons/base/models/res_users.py

```diff
@@ -193,10 +193,13 @@ def SELF_WRITEABLE_FIELDS(self):
     def _default_groups(self):
         """Default groups for employees
 
-        All the groups of the Template User
+        All the groups of the Default User Group
         """
-        default_user = self.env.ref('base.default_user', raise_if_not_found=False)
-        return default_user.sudo().group_ids if default_user else []
+        groups = self.env.ref('base.group_user', raise_if_not_found=False)
```

Comment:
Always exists, otherwise we have a problem.
```suggestion
        groups = self.env.ref('base.group_user')
```